docs/en/operation/Server-Logging.md

```diff
@@ -29,7 +29,9 @@ By default Alluxio processes' log files can be found under `${ALLUXIO_HOME}/logs
 ### Configuring the Log Location 
 
 The location of the logs is determined by the `alluxio.logs.dir` property. This is a JVM
-property which cannot be set in the `alluxio-site.properties` file. See the
+property which cannot be set in the `alluxio-site.properties` file. 
+TODO: if stating where the property cannot be set, please state where it CAN be set
```

Comment:
-The location of the logs is determined by the `alluxio.logs.dir` property. This is a JVM
-property which cannot be set in the `alluxio-site.properties` file.
-TODO: if stating where the property cannot be set, please state where it CAN be set
+The location of the logs is determined by the `alluxio.logs.dir` property. This can only be set via JVM
+property; it cannot be set in the `alluxio-site.properties` file.